lib/runners/cloud-hypervisor.nix

```diff
@@ -65,6 +65,9 @@ let
   };
 
 in {
+  hypervisor = "crosvm";
+  executable = pkgs.crosvm;
```

Comment:
You called this `package` elsewhere. I'm not sure if this is a very descriptive name, either.

Reply:
gah i renamed it to package because i thought executable was a rubbish name and messed this bit up. 

I can just drop the commit, yeah. What I want is to be able to nix build something and get the modified qemu that will actually be used. Currently it only exists as an interpolation in a string so it's annoying to build the qemu to look at it.

Reply:
I added `passthru.hypervisor` in f92c945. Do you still need something along `hypervisorExe`? For debugging purposes only?

Reply:
That change is buggy since it gives what the user configured in all cases, rather than what the script is generated for:

```
Loading installable 'nixosConfigurations.boxMicro.config.microvm'...
Added 28 variables.
nix-repl> runner
runner
nix-repl> runner.
runner.cloud-hypervisor  runner.crosvm            runner.firecracker       runner.kvmtool           runner.qemu              runner.stratovirt
nix-repl> runner.crosvm.hypervisor
"qemu"
```

Also, yeah, I would like some attribute that is the hypervisor derivation.

Reply:
Fixed in c28a0e0